package.json

```diff
@@ -106,14 +106,19 @@
     "angular-cookies": "^1.7.9",
     "angular-route": "^1.7.9",
     "bootstrap": "^3.4.1",
+    "create-react-class": "15.6.2",
```

Comment:
Why is this added?

Reply:
Discussed in this [comment](https://github.com/oppia/oppia/pull/9352#discussion_r429879484).